docs/language/overview.md

```diff
@@ -1763,37 +1739,36 @@ To unify disparate data sources, data is often cleaned up to fit into
 a well-defined set of schemas, which combines the data into a unified
 store like a data warehouse.
 
-In Zed, this cleansing process is called "shaping" the data and Zed leverages
-its rich, super-structured type system to perform core aspects of
-data transformation.
+In Zed, this cleansing process is called "shaping" the data, and Zed leverages
+its rich, [super-structured](../formats/README.md#2-zed-a-super-structured-pattern)
+type system to perform core aspects of data transformation.
 In a data model with nesting and multiple scalar types (such as Zed or JSON),
-shaping includes converting type of leaf fields, adding or removing fields
+shaping includes converting the type of leaf fields, adding or removing fields
 to "fit" a given shape, and reordering fields.
 
 While shaping remains an active area of development, the core functions in Zed
 that currently perform shaping are:
 
-* [cast](functions/cast.md) - coerce a value to a different type
-* [crop](functions/crop.md) - remove fields from a value that are missing in a specified type
-* [fill](functions/fill.md) - add null values for missing fields
-* [order](functions/order.md) - reorder record fields
-* [shape](functions/shape.md) - apply cast, fill, and order
+* [`cast`](functions/cast.md) - coerce a value to a different type
+* [`crop`](functions/crop.md) - remove fields from a value that are missing in a specified type
+* [`fill`](functions/fill.md) - add null values for missing fields
+* [`order`](functions/order.md) - reorder record fields
+* [`shape`](functions/shape.md) - apply `cast`, `fill`, and `order`
 
 They all have the same signature, taking two parameters: the value to be
 transformed and a type value for the target type.
 
 > Another type of transformation that's needed for shaping is renaming fields,
-> which is supported by the [rename operator](operators/rename.md#operator).
-> Also, the [yield operator](operators/yield.md#operator)
+> which is supported by the [`rename` operator](operators/rename.md).
+> Also, the [`yield` operator](operators/yield.md)
 > is handy for simply emitting new, arbitrary record literals based on
 > input values and mixing in these shaping functions in an embedded record literal.
-> The [fuse aggregate function](aggregates/fuse.md) is useful for fusing
-> values into a common schema within a group-by schema though a type is returned
-> rather than values.
+> The [`fuse` aggregate function](aggregates/fuse.md) is also useful for fusing
+> values into a common schema, though a type is returned rather than values.
```

Comment:
Please review this one closely. The original sentence seemed confusing and I thought it could be simplified, but I want to make sure I didn't wreck its essence.

Reply:
@mccanne: You should probably weigh in here.

Reply:
LGTM